caikit/runtime/service_generation/create_service.py

```diff
@@ -18,17 +18,16 @@
 
 # Standard
 from enum import Enum
-from typing import List, Type
+from typing import Dict, List, Type
 
 # First Party
 import alog
 
 # Local
 from ... import get_config
-from .core_module_helpers import get_module_info
```

Comment:
I think these were the only usages of `get_module_info`, right?
If so, mind deleting the function too?